docs/server-side/v2/api-reference/room-codes/get-room-code.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: Retrieve Room Codes
+nav: 4.14
+---
+
+Retrieves room codes for all roles in a Room. You can also retrive room codes for a specific role or only enabled room codes using these arguments in the endpoint. 
+
+<GetRequest title="https://api.100ms.live/v2/room-codes/room/<room_id>" />
+<Request id="req-comp-0">
+
+```bash
+curl --location --request GET 'https://api.100ms.live/v2/rooms' \
+--header 'Authorization: Bearer <management_token>' 
+```
+</Request>
+   
+<ResponseBox id="resp-0" status="200 OK">
+```json
+    {
+    	limit: 20,
+    	data: [
+    		{
+    			code: "hkj-sdf-fda",
+    			room_id: "new-room-1662723668",
+    			role: "host",
+    			enabled: true,
+    			created_at: <timestamp>,
+    			updated_at: <timestamp>,
+    		},
+    		{
+    			code: "wiw-giw-bzs",
+    			room_id: "new-room-1662723668",
+    			role: "guest",
+    			enabled: true,
+    			created_at: <timestamp>,
+    			updated_at: <timestamp>,
+    		}
+    	],
+    	last: ""
+    }
+    ```
+</ResponseBox>    
+ 
+### Arguments 
```

Comment:
these are query parameters right? i think we should call them out as "query parameters"

Reply:
these are called out in Room Code Object, otherwise we will have to callout the Arguments in each API page